policyengine/utils/charts.py

```diff
@@ -32,6 +32,7 @@ def add_fonts():
 WHITE = "#FFFFFF"
 TEAL_98 = "#F7FDFC"
 BLACK = "#000000"
+LIGHT_LIGHT_GRAY = "#F4F4F4"
```

Comment:
__nitpick, non-blocking:__ Rename this `FOG_GRAY`

Suggested by Claude. We just have so many `GRAY`s.

Reply:
Done